packages/p2p/src/pages/buy-sell/buy-sell-form.scss

```diff
@@ -53,10 +53,12 @@
             gap: 0.5rem;
 
             &--floating {
-                position: fixed;
+                position: absolute;
 
-                @include mobile {
-                    position: absolute;
+                &:hover {
+                    @include desktop {
+                        position: fixed;
```

Comment:
position fixed is what caused the issue since it makes it stick to the page on scroll. But position fixed is the only position that allows the tooltip to hover over the modal header, and because user only sees the tooltip on hover, then we set the position to fixed